sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -53,47 +60,253 @@ def _is_ipython(self):
+                            exp_object.metadata.name,
+                            namespace,
+                            exp_object.metadata.name,
+                        )
+                    )
+                )
+
+    def tune(
+        self,
+        # TODO (andreyvelich): How to be consistent with other APIs (name) ?
+        name: str,
+        objective: Callable,
+        parameters: Dict[str, Any],
+        base_image: str = constants.BASE_IMAGE_TENSORFLOW,
+        namespace: str = utils.get_default_target_namespace(),
+        algorithm_name: str = "random",
+        objective_metric_name: str = None,
+        additional_metric_names: List[str] = [],
+        objective_type: str = "maximize",
+        objective_goal: float = None,
+        max_trial_count: int = None,
+        parallel_trial_count: int = None,
+        max_failed_trial_count: int = None,
+        retain_trials: bool = False,
+        packages_to_install: List[str] = None,
+        pip_index_url: str = "https://pypi.org/simple",
+    ):
+        """Create HyperParameter Tuning Katib Experiment from the objective function.
+
+        Args:
+            name: Name for the Experiment.
+            objective: Objective function that Katib uses to train the model.
+                This function must be Callable and it must have only one dict argument.
+                Katib uses this argument to send HyperParameters to the function.
+            parameters: Dict of HyperParameters to tune your Experiment. You
+                should use Katib SDK to define the search space for these parameters.
+
+                For example: `parameters = {"lr": katib.double(min=0.1, max=0.2)}`
+
+                Also, you can use these parameters to define input for your
+                objective function.
+            base_image: Image to use when executing the objective function.
+            namespace: Namespace for the Experiment.
+            algorithm_name: Search algorithm for the HyperParameter tuning.
+            objective_metric_name: Objective metric that Katib optimizes.
+            additional_metric_names: List of metrics that Katib collects from the
+                objective function in addition to objective metric.
+            objective_type: Type for the Experiment optimization for the objective metric.
+                Must be one of `minimize` or `maximize`.
+            objective_goal: Objective goal that Experiment should reach to be Succeeded.
+            max_trial_count: Maximum number of Trials to run.
+            parallel_trial_count: Number of Trials that Experiment runs in parallel.
+            max_failed_trial_count: Maximum number of Trials allowed to fail.
+            retain_trials: Whether Trials' resources (e.g. pods) are deleted after Succeeded state.
+            packages_to_install: List of Python packages to install before
+                executing the objective function.
```

Comment:
how about saying that these field is for packages except already installed in the `base_image` ?

Reply:
Sure. @anencore94 What do you think about this:
```
List of Python packages to install in addition to base image packages. These packages are installed before executing the objective function.
        ```

Reply:
Looks nice :) Thanks